README.md

```diff
@@ -0,0 +1,46 @@
+# HGCTPGEmulator
+
+## Overview
+
+Ensemble of python and C++ modules to run the HGC-TPG backend firmware emulator.
+
+## Dependencies
+
+The testing framework uses the `nlohmann_json` library, which must be installed and accessible. If not, it can be easily set, e.g. in a conda environment:
+
+```
+conda create --name newEnv
+conda activate newEnv
+conda install -c conda-forge nlohmann_json pkg-config
+```
+
+`pkg-config` is also needed inside the conda environment, in order to automatically pick up the package location inside the Makefile.
+
+
+## Stage1 emulator
+
+Temporary input and config files are available at the following link:
+
+https://cernbox.cern.ch/index.php/s/mLpzkHbba8dXgDD
+
+The stage 1 emulator can be tested with two different formats of inputs, as provided here:
+
+- `inputs/inputs.txt`: default input format used in S2 emulation.
+
+  These files directly contain the TC information needed to run the emulator.
+
+- `inputs/TClist.txt`: list generated from `ModXXXX_stimuli.txt` files, containing input TCs per module, as used for S1 firwmare testing. 
+
+  For these, additional information on the mapping of FPGA/modules/TCs in <img src="https://render.githubusercontent.com/render/math?math=(r/z,\phi)"> is needed. These are `.csv` files, generated with `python/generateTCMap.py`.
```

Comment:
Do the files in the python directory belong in this repo, or are they pulled in from somewhere else?

Reply:
`load_mappings.py` is taken from the S1 firmware generator repository. The other two scripts are specific to `HGCTPGEmulator`, but depends on the former script. However all these scripts will in principle not be useful anymore once we settle on the parameter extraction in CMSSW.